tests/test_runner/test_discover_runner.py

```diff
@@ -210,3 +211,19 @@ def count_tests(**kwargs):
         self.assertEqual(count_tests(tags=['foo'], exclude_tags=['bar']), 2)
         self.assertEqual(count_tests(tags=['foo'], exclude_tags=['bar', 'baz']), 1)
         self.assertEqual(count_tests(exclude_tags=['foo']), 0)
+
+    def test_display_tags(self):
+        """
+        Test that tags are displayed for verbosity level greater than or equal to 2
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).